docs/get-started/try-viam/rover-resources/rover-tutorial-2.md

```diff
@@ -0,0 +1,383 @@
+For both 18650 and RC-type battery inputs, the nominal voltage is 14.8V, so you should set the low voltage threshold to 14.7.
+You can adjust this value if using a battery that has an alternative nominal voltage.
+
+To set the low voltage cutoff and reconnect voltage:
+
+1.  Turn on the circuit using the switch.
+    The LED indicater should indicate a current voltage level of 14.8-16V depending on the battery charge status:
+
+    {{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/circuit-led.png" resize="300x" declaredimensions=true alt="LED with voltage level displayed on low voltage cutoff circuit" >}}
+
+2.  Hold down the left button until the LED display starts flashing with the low cutoff value.
+    The factory default low cutoff value is 12V.
+3.  Use the left (+) and right (-) buttons to set the voltage to 14.7V (or whatever you want the cutoff to be).
+4.  Wait for the indicator to stop flashing.
+5.  Hold down the right button until the LED display starts flashing with the reconnect voltage value. The factory default reconnect voltage is 2.0V.
+6.  Use the left (+) and right (-) buttons to set the reconnect voltage to 0.2V.
+7.  Wait for the indicator to stop flashing.
+
+Your voltage cutoff circuit is now configured.
+When the voltage drops below 14.8V (or reaches the cutoff point you chose), a relay will disconnect the motherboard.
+A minimum voltage of 14.9V will be needed to reconnect the power (that is, after charging the batteries).
+
+### Connect the ribbon cable, Pi, and camera
+
+{{% alert title="Tip" color="tip" %}}
+If you are using another board, follow the instructions in [Alternative board configurations](#alternative-board-configurations) in place of this step.
+{{% /alert %}}
+
+To be able to attach the Raspberry Pi, unscrew the top of the rover with the biggest Allen key.
+Then use the smallest Allen key and the provided M2.5 screws to attach the Raspberry Pi to your rover through the standoffs on the motherboard.
+The Raspberry Pi 4 should be mounted such that the USB ports are to the right, as viewed from above.
+
+Use the ribbon cable to connect the Raspberry Pi 4 to the motherboard.
+The ribbon cable comes connected to the motherboard out of the box, wrap it over the top of the Raspberry Pi 4 and connect with the GPIO pins as shown:
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/ribbon-cable1.png" resize="400x" declaredimensions=true alt="Ribbon cable attached to pi" >}}
+<br>
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/ribbon-cable2.png" resize="400x" declaredimensions=true alt="Ribbon cable attached to pi" >}}
+
+Also, connect the webcam's USB lead to any USB port on your Pi.
+
+Assuming you are using a Raspberry Pi 4, you can skip the following section and move to [Screw the top plate back on and switch the rover on](#screw-the-top-plate-back-on-and-switch-the-rover-on).
+
+#### Alternative board configurations
```

Comment:
I think as you have it makes sense but I don't like that it's not in the `on this page section ` then - how do you feel about h3?
```suggestion
### Alternative board configurations
```